sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/ModelsRepositoryAsyncClient.java

```diff
@@ -62,4 +63,64 @@
     public ModelsRepositoryServiceVersion getServiceVersion() {
         return this.serviceVersion;
     }
+
+    /**
+     * Gets a collection of model definitions.
+     *
+     * @param dtmi A well-formed DTDL model Id. See <a href="https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md">DTDL specs</a>. For example 'dtmi:com:example:Thermostat;1'.
+     * @return A Map containing the model definition(s) where the key is the dtmi
+     * and the value is the raw model definition string.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Map<String, String>> getModels(String dtmi) {
+        return getModels(dtmi, this.defaultDependencyResolutionOption);
+    }
+
+    /**
+     * Gets a collection of model definitions.
+     *
+     * @param dtmi                 A well-formed DTDL model Id. See <a href="https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md">DTDL specs</a>. For example 'dtmi:com:example:Thermostat;1'.
```

Comment:
Why so much spacing here after `dtmi`?

Reply:
That space is Intelij auto formatting feature that lines up all parameter descriptions together

Reply:
I have since fixed this as the azure core build plugins also complain about this.